src/bp-members/classes/class-bp-members-invitations-feature.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * BuddyPress Members Invitations feature Class.
+ *
+ * @package BuddyPress
+ * @subpackage Members
+ * @since 15.0.0
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * The BP Members Invitations feature is optional.
+ *
+ * If you want to disable it, you can use:
+ * `add_filter( 'bp_is_members_invitations_active', '__return_false' );`
+ *
+ * @since 15.0.0
+ */
+class BP_Members_Invitations_Feature extends BP_Component_Feature {
+
+	/**
+	 * Invitations Feature initialization.
+	 *
+	 * @since 15.0.0
+	 */
+	public function __construct() {
+		parent::init( 'invitations', 'members' );
+	}
+
+	/**
+	 * Set up Signups feature global variables.
+	 *
+	 * @since 15.0.0
+	 */
+	public function globals() {
+		buddypress()->members->invitations = new stdClass();
```

Comment:
This class seems to be only used to store the `query_loop` as used in `bp-members-template.php`. Would it make more sense to fold it into the `members->active_features->invitations` object in the future?